cfgen/cfgen

```diff
@@ -1436,53 +1453,79 @@ def pool_drives_with_nodes(m0conf: Dict[Oid, Any]) -> List[Tuple[Oid, Oid]]:
     return all_drives
 
 
+def pool_drives_from_pooltypes(
+                cluster_desc: Dict[str, Any],
+                pool_desc: Dict[str, Any]):
+
+    for pool in cluster_desc['pools']:
+        ptype = pool_type(pool_desc)
+        if ptype is PoolT.sns:
+            return(get_pool_diskrefs(pool_desc))
+        if ptype is PoolT.dix:
+            return(get_pool_diskrefs(pool_desc))
```

Comment:
what disk_refs are returned for dix pool?

Reply:
It's returning `None` for DIX pools.
`[root@ssc-vm-g2-rhev4-1947 hare]# hctl bootstrap --mkfs ~/1n.yaml
2021-12-15 23:48:41: Generating cluster configuration...pool names: pool1
pooltypes: PoolT.sns
disk_refs in sns: [{'path': '/dev/sdb', 'node': 'ssc-vm-g2-rhev4-1947.colo.seagate.com'}, {'path': '/dev/sdc', 'node': 'ssc-vm-g2-rhev4-1947.colo.seagate.com'}]
pool names: pool2
pooltypes: PoolT.dix
disk_refs in dix: None
`